sdk/python/setup.py

```diff
@@ -91,13 +91,14 @@ def read_readme() -> str:
         'Programming Language :: Python :: 3.10',
         'Programming Language :: Python :: 3.11',
         'Programming Language :: Python :: 3.12',
+        'Programming Language :: Python :: 3.13',
         'Topic :: Scientific/Engineering',
         'Topic :: Scientific/Engineering :: Artificial Intelligence',
         'Topic :: Software Development',
         'Topic :: Software Development :: Libraries',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
-    python_requires='>=3.9.0,<3.13.0',
+    python_requires='>=3.9.0',
```

Comment:
did we want to do the same for the kubernetes platform package? 
https://github.com/kubeflow/pipelines/blob/master/kubernetes_platform/python/setup.py#L79

Reply:
Good point! Will remove that as well.